ql/src/test/results/clientpositive/llap/lateral_left_semi_join.q.out

```diff
@@ -0,0 +1,305 @@
+Warning: Shuffle Join MERGEJOIN[54][tables = [sq_1_notin_nullcheck]] in Stage 'Reducer 3' is a cross product
```

Comment:
One of the parent of `Reducer 3` is a union work(alias `T`), when CrossProductHandler [analyzes](https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/optimizer/physical/CrossProductHandler.java#L280-L289) the union work, as `work.getAllRootOperators()` returns an empty set, so the inputs of  `t` and `lv` do not added to the reduce sink info of the join, cause the waring message missing some table aliases.

Reply:
Using the MR, the warning message is `Warning: Shuffle Join JOIN[34][tables = [lv, t, sq_1_notin_nullcheck]] in Stage 'Stage-1:MAPRED' is a cross product`.